### DISCUSSION TITLE
Gatsby SSR

### DISCUSSION THREAD

**Discussion by tnorlund** [gatsbyjs/gatsby#28113] [Help]
So I did a clean install of Mac OS, and I thought it'd be a good time to refactor.

I'm currently trying to implement SSR, but when I `Gatsby develop`, I get this error:
```bash
error Generating SSR bundle failed
Can't resolve 'components/utils/constants' in '/Users/tnorlund/tutorial-part-four'
```

How do I get SSR to see the `/src/` directory?

**Comment by KyleAMathews** [comment]
It looks like you're trying to require a file that doesn't exist?
